test/controllers/api/authorizations_controller_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+describe Api::AuthorizationsController do
+  let (:user) { create(:user) }
+  let (:token) { OpenStruct.new.tap do |t|
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.